### ISSUE TITLE
Examples with Ref not parsed when in responses

### ISSUE THREAD

**Issue by JehandadK** [swagger-api/swagger-parser#837]
Simply put

```yaml
responses:
  content:
    examples:
       key:
         $ref: 'file.yaml#components/examples/key'

```

Does not get resolved.

**Comment by ymohdriz** [comment]
Hi @JehandadK 

Can you please provide a sample open api file to reproduce this issue?

Thanks,
Mohammed Rizwan

**Comment by gracekarina** [comment]
Fixed by #838
